modules/plugin/jdbc/jdbc-postgis/src/main/java/org/geotools/data/postgis/PostgisNGDataStoreFactory.java

```diff
@@ -79,7 +79,7 @@
             "Due to differences in the type systems the result might not be the same as evaluating " +
             "them in memory, including the SQL failing with errors while the in memory version works fine. " +
             "However this allows to push more of the filter into the database, increasing performance." +
-            "the postgis table.", false, new Boolean(false),
+            "the postgis table.", false, new Boolean(true),
```

Comment:
This is an intended changed ?

Reply:
Yes, it's written in the Jira ticket (flip the default to encode functions)